src/main/java/uk/gov/justice/digital/delius/data/api/ContextlessAppointmentCreateRequest.java

```diff
@@ -17,17 +17,28 @@
 
     @NotNull
     @ApiModelProperty(required = true)
-    private OffsetDateTime appointmentStart;
+    private String contractType;
 
     @NotNull
     @ApiModelProperty(required = true)
-    private OffsetDateTime appointmentEnd;
+    private OffsetDateTime referralStart;
```

Comment:
Is it worth grouping the context and detail of the contextless requests using a couple of comments?

Reply:
Sounds sensible to me - will do